Source/FakeItEasy.Tests/FakeOptionsBuilderTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace FakeItEasy.Tests
+{
+    using System;
+    using FakeItEasy.Creation;
+    using FluentAssertions;
+    using NUnit.Framework;
+    using TestHelpers;
+
+    public class FakeOptionsBuilderTests

```

Comment:
Ah, I wasn't aware of that! No, I'm not concerned. If it works without the attribute, it's fine by me.